src/polyfills/cpus.ts

```diff
@@ -5,5 +5,5 @@ import {
 
 export const availableParallelism = (): number =>
   typeof nodeAvailableParallelism === 'function'
-    ? Math.floor(nodeAvailableParallelism() / 2)
+    ? nodeAvailableParallelism()
     : (cpus()?.length ?? 1);
```

Comment:
Not 100% sure `1` should be set here, because later it gets divided by `2`. `Math.floor()` turns it into zero. Maybe it could be `0` from very beginning?

Reply:
Hm, `Math.floor(0.5)` will be `0`.

Reply:
Yep. Not sure if your intention was to have `0` or `1` as the final fallback value.

Reply:
After #603, the idea is to allows `0` only by setting it manually.

Reply:
Perhaps `Math.round()` should be used?

Reply:
> Perhaps `Math.round()` should be used?

Yes, good ideia. It would solve the problem of rounding to 0.